generate_docker.py

```diff
@@ -55,8 +55,8 @@ def generate_dockerfile(base_image_name: str) -> None:
         with open(dockerfile_name, 'w') as out_file:
             for line in template:
                 if line.startswith('LABEL maintainer='):
-                    out_file.write("LABEL maintainer=\"{0} <{1}>\"\n".format(configuration.AUTHOR_NAME,
-                                                                             configuration.AUTHOR_EMAIL))
+                    out_file.write("LABEL maintainer=\"{0} <{1}>\"\n".format(ConfigurationInfo.author_name,
+                                                                             ConfigurationInfo.author_email))
                 elif line.startswith('FROM base-image'):
                     out_file.write("FROM {0}\n".format(base_image_name))
                 else:
```

Comment:
When the Dockerfile is generated, and then I try to build an image, there is a permissions error for transformer.py. I tried to solve this by adding in the permissions/changing lines from plot-base-rgb, but they did not solve the problem. Output is attached
<img width="1315" alt="Screen Shot 2020-07-23 at 9 00 31 AM" src="https://user-images.githubusercontent.com/62153719/88309334-06068100-ccc3-11ea-8dd5-d5c81cb5025f.png">
<img width="1390" alt="Screen Shot 2020-07-23 at 9 01 35 AM" src="https://user-images.githubusercontent.com/62153719/88309416-220a2280-ccc3-11ea-8808-3fc5c2634d78.png">

Reply:
There were issue with the Dockerfile.template file that have now been resolved.